### DISCUSSION TITLE
Welcome to SRAD_Avionics Discussions!

### DISCUSSION THREAD

**Discussion by varun-un** [Terrapin-Rocket-Team/SRAD-Avionics#102] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by varun-un** [comment]
> ## 👋 Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Wow what a great introduction you definitely wrote and was not autogenerated. I'll start! I'm Varun, one of the avionics subteam's biggest fans, and I hope to learn a lot and make a really good flight computer!

**Comment by DrewBrandt** [comment]
am drew
